SkEditor/Utilities/Docs/SkUnity/SkUnityProvider.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using SkEditor.API;
+
+namespace SkEditor.Utilities.Docs.SkUnity;
+
+public class SkUnityProvider : IDocProvider
+{
+    private const string BaseUri = "https://api.skunity.com/v1/%s/docs/";
+
+    private readonly HttpClient _client = new HttpClient()
+        .WithUserAgent("C# App");
```

Comment:
user agent changes pretty much nothing, but why `C# App`? 😛 I would change it to SkEditor